internal/globalMerge.ecl

```diff
@@ -155,13 +155,13 @@ EXPORT globalMerge := MODULE
               id = id - lastgroupend;
               pid = pid - lastgroupend;
               *(uint32_t *)pos = wi;
-              pos += sizeof(uint32_t);
+              pos =(uint32_t *)pos + sizeof(uint32_t);
```

Comment:
This does not mean the same thing.  By casting it, all the offsets will be scaled by 4.